_posts/2020-09-16-cockpit-228.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 228
+author: kkoukiou
+date: 2020-09-16
+tags: cockpit linux
+slug: cockpit-228
+category: release
+summary: Cockpit
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 228.
+
+### Accounts: Allow setting weak passwords
+
+Cockpit now allows users to set up a weak password. The user gets notified that
+the password is weak, but clicking the submit button again will use the selected password anyway.
+This is similar to how weak passwords are handled in the Anaconda installer.
+
+![Accounts weak password](/images/accounts-weak-password.png){:.screenshot}
+
+### Changes to remote host logins
+
+Cockpit now never tries to log into remote hosts with your initial login password,
+and consequently does not promote password reuse across accounts anymore.
+Instead, Cockpit now helps with setting up SSH keys for users that want automatic
+and password-less login to remote hosts.
+
+Here is a quick [demo](https://www.youtube.com/watch?v=4UBZWKk-T28).
```

Comment:
Let's also show the screenshot to give people a quick idea of what this is about.